api/src/convert/verification_report.rs

```diff
@@ -3,15 +3,14 @@
 //! Convert to/from external::VerificationReport
 
 use crate::external;
-use mc_attest_core::{VerificationReport, VerificationSignature};
-use protobuf::RepeatedField;
+use mc_attest_verifier_types::{VerificationReport, VerificationSignature};
```

Comment:
To sanity check - the mc_attest_verifier_types crate is being created to make it easier for to access the AVR data right? What's the benefit of moving it?

Reply:
James pointed out that `mc-attest-core` has a large footprint, and while it's already included in the enclave, it would also get included in all clients that use `mc-transaction-core` and by extension `mc-api`.

Reply:
Should we perhaps also create a separate PR that puts this into main as well?

Reply:
It seems OK to defer that till we merge this feature branch. LMK if you disagree.